### DISCUSSION TITLE
error 404

### DISCUSSION THREAD

**Discussion by fbollon** [MohmmedAshraf/laravel-translations#60] [Q&A]
Hello,

I'm trying to use this package in a Laravel project whose APP_URL=http://localhost:82/myapp

I have several applications on the same server and each application is in a folder at the root of the web server

when I try to access http://localhost:82/myapp/translations

I get a blank page with 404 errors 
http://localhost:82/vendor/translations-ui/assets/app-6e494c7d.js
http://localhost:82/vendor/translations-ui/assets/app-d264aee5.css

because the "myapp" folder is missing

is there a configuration to correct this problem?

Thanks

**Comment by Muhammadislom** [comment]
Yes, the issue arises because Laravel's translations-ui package is serving assets from the /vendor/translations-ui/assets/ path, but your application is located in a subdirectory (/myapp). You need to adjust the asset URLs so they include the /myapp prefix. Here’s how you can fix it:

ASSET_URL=http://localhost:82/myapp

If the package allows it, modify how assets are loaded by using:

<script src="{{ asset('vendor/translations-ui/assets/app-6e494c7d.js') }}"></script>
<link rel="stylesheet" href="{{ asset('vendor/translations-ui/assets/app-d264aee5.css') }}">


If the assets are not loading properly, try running:

php artisan storage:link
